adding_species_manuscript.tex

```diff
@@ -540,24 +546,21 @@ \subsection*{Basic setup for chromosome-level simulations}
 These five categories of parameters are sufficient for generating simulations
 under neutral evolution. Such simulations are useful for a number of purposes,
 but they cannot be used to model the influence of natural selection on patterns of genetic variation.
```

Comment:
removed this because it appears to be redundant with previous discussion. The main point here is how to model selection and not so much why it is important